#!/usr/bin/env python
PACKAGE = "mantis_controller_id"

from dynamic_reconfigure.parameter_generator_catkin import *
import math

gen = ParameterGenerator()

gen.add("wait_for_path", bool_t, 0, "Sets the controller to wait for trajectory input before starting", True)
gen.add("track_end", bool_t, 0, "Enables end effector tracking (instead of robot base)", False)
gen.add("track_j2", bool_t, 0, "Enables 'joint 2' tracking (instead of robot base)", False)
gen.add("accurate_z_tracking", bool_t, 0, "", False)
gen.add("accurate_end_tracking", bool_t, 0, "", False)
gen.add("reference_feedback", bool_t, 0, "Enables internal reference feedback to be sent", False)

exit(gen.generate(PACKAGE, "controller_id_node", "ControlParams"))
